keycloak.d.ts

```diff
@@ -192,7 +192,20 @@ declare namespace KeycloakConnect {
     isExpired(): boolean
   }
 
-  type GaurdFn = (accessToken: string, req: express.Request, res: express.Response) => boolean
+  type GaurdFn = (accessToken: Token, req: express.Request, res: express.Response) => boolean
+
+  interface EnforcerOptions {
+    response_mode?: string,
+    resource_server_id?: string,
+    claims?: (...args: any[]) => any
+  }
+
+  interface AuthZRequest {
```

Comment:
I think `claim_token_format` is missing here per:
https://github.com/keycloak/keycloak-nodejs-connect/blob/043807542cb94d5477da867808e9b5b559f9eddf/middleware/auth-utils/grant-manager.js#L123

Reply:
You are right. I added it and will push the commit in a few minutes. Any idea when this is going to be released?